src/sys/unix.rs

```diff
@@ -1901,6 +1966,39 @@ impl crate::Socket {
         Ok(NonZeroU32::new(index))
```

Comment:
Shouldn't `device_index` also be renamed?

Reply:
Done.